api_inference_community/routes.py

```diff
@@ -35,6 +37,7 @@ async def pipeline_route(request: Request) -> Response:
             sampling_rate = pipe.sampling_rate
         except Exception:
             sampling_rate = None
+        mime_types = validate_mime_types(request.headers["accept"])
```

Comment:
Ok, my suggestion:

We can add a new `parameter` which would be `audio_format` maybe ? (There might be a  better name).

We can whitelist it to be `flac`, `mp3`, `ogg`, `wav` so that we cover most used formats I think.

That means users can choose encoding directly from params. We can then respect `headers["accept"]` to override the parameter (if it hasn't been specified).

Because it's a parameter, there's much less code change in other places.

Reply:
We could even make it more generic and just have optional `format` for any pipeline